packages/fx-core/src/component/driver/file/createOrUpdateEnvironmentFile.ts

```diff
@@ -70,8 +70,12 @@ export class CreateOrUpdateEnvironmentFileDriver implements StepDriver {
       );
       context.logProvider?.debug(`Updated envs: ${JSON.stringify(updatedEnvs)}`);
       await fs.writeFile(target, updatedEnvs.join(os.EOL));
+      const output = new Map<string, string>();
+      for (const [key, value] of Object.entries(args.envs)) {
+        output.set(key, value);
+      }
       return {
-        output: new Map<string, string>(),
```

Comment:
This change has side-effect - if I'm updating my own `.env` file, e.g., `some-runtime.env`, those variables will be injected into ttk process as well.

To directly modify ttk's env, how about using `::set-output` script action?